### DISCUSSION TITLE
404 is the only response received

### DISCUSSION THREAD

**Discussion by Itzdlg** [shlinkio/discussions#54] [Help wanted] [answered]
#### How Shlink is set-up

* Shlink Version: 2.10.2
* PHP Version: 8.0
* How do you serve Shlink: self-hosted nginx
* Database engine used: mariadb

#### Summary

I've setup shlink on my VPS with a new domain oversized.link, and it had and still is also using schooltests.me for other applications. After setting up nginx, I was able to go to the site oversized.link to get the beautiful 404 page, which is expected since it's the base site. However, if I try to visit the /rest/health endpoint, I get a different 404 (not the nicely styled one). If I try to visit a short-url, then I still get a 404 (the nicely styled one). There aren't any errors in data/log files and NGINX access logs are bland. If you want to try it out, there's a short-url QG7Q0 (https://oversized.link/QG7Q0). As a bonus (although this was probably already inferred), I'm unable to connect to the site through the https://app.shlink.io/ web-app.

![image](https://user-images.githubusercontent.com/15074893/149272684-039628f3-75ff-4886-b3b1-83d4df542fe9.png)

If there's anything else I can provide to try and solve this please let me know. Thank you in advance!

**Comment by acelaya** [comment]
Can you share your nginx config?

**Reply by Itzdlg** [reply]
```
server {
    server_name oversized.link;
    listen 443 ssl;
    root /etc/shlink/public;
    index index.php;
    charset utf-8;

    ssl_certificate /etc/letsencrypt/active/oversized-link/fullchain.pem;
    ssl_certificate_key /etc/letsencrypt/active/oversized-link/privkey.pem;

    access_log /var/log/nginx/shlink.log;

    location / {
        try_files $uri $uri/ /index.php$is_args$args;
    }

    location ~ \.php$ {
        fastcgi_split_path_info ^(.+\.php)(/.+)$;
        fastcgi_pass unix:/var/run/php/php8.0-fpm.sock;
        fastcgi_index index.php;
        include fastcgi.conf;
    }

    location ~ /\.ht {
        deny all;
    }
}
```

**Reply by Itzdlg** [reply]
If it means anything, before I had setup shlink through the dist package, I had attempted to use the docker installation. That wasn’t working for me because I had to port map to change it from 8080 & use host network for the database, which doesn’t work unfortunately. I had also deleted the config/params/generated_params file before reconfiguring with bin/install. Perhaps this caused some sort of problem? My config has the url_shortener domain option as oversized.link and the router base_path is empty. Could those not be the correct values for my environment?

**Comment by Itzdlg** [comment, marked as answer]
I've completely deleted the database and installation, and then reinstalled shlink. It now seems to work. I'd just like to say thank you for creating & maintaining this OSS project and for the extremely quick response earlier!

**Reply by acelaya** [reply]
Oh, nice. I still had to come back to this, but I'm glad it's working now.

And thanks for the kind words :)
